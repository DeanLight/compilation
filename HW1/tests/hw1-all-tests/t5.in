%literals
obj endobj
[]][
>><<null
true	false
