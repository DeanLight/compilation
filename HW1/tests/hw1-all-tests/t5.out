1 COMMENT %literals
2 OBJ obj
2 ENDOBJ endobj
3 LBRACE [
3 RBRACE ]
3 RBRACE ]
3 LBRACE [
4 RDICT >>
4 LDICT <<
4 NULL null
5 TRUE true
5 FALSE false
6 EOF 
